WebChannelAPI.md

```diff
@@ -0,0 +1,474 @@
+# Meld Studio's WebChannel API Documentation
+
+## Overview
+
+Meld Studio's WebChannel API enables JavaScript clients to interact with various components of Meld Studio. It allows for the management of `scenes`, `audio tracks`, `effects`, and other `session` states, providing an RPC-like link between the web clients and the application's backend.
+
+### Qt's WebChannel Protocol
+
+Meld Studio's WebSocket API is managed by a `QWebChannel` protocol server. This is used to provide a Qt-like API in Javascript. 
+
+A usable Javascript `QWebChannel` client protocol implementation can be found here: https://cdn.jsdelivr.net/npm/qwebchannel@6.2.0/qwebchannel.min.js
+
+See [Qt's Documentation](https://doc.qt.io/qt-6/qtwebchannel-javascript.html) for more details.
+
+### Quickstart Code Sample
+
+This code block will get you up and communicating with Meld Studio.
+
+```js
+ function connect() {
```

Comment:
Might want to clarify that this example runs in the browser (but Node is likely supported with some tweaks)

Reply:
Fixed in 7d676dc136ad0ef8390ce78eeca8b749bd479623